specification/computeschedule/ComputeSchedule.Management/examples/2024-10-01/ScheduledActions_VirtualMachinesGetOperationStatus.json

```diff
@@ -1,13 +1,13 @@
 {
-  "title": "ScheduledActions_VirtualMachinesGetOperationStatus_MaximumSet_Gen - generated by [MaximumSet] rule",
+  "title": "ScheduledActions_VirtualMachinesGetOperationStatus_MaximumSet_Gen",
```

Comment:
can we also remove _MaximumSet_Gen from titles e.g.-
 "title": "ScheduledActions_VirtualMachinesGetOperationStatus_MaximumSet_Gen",